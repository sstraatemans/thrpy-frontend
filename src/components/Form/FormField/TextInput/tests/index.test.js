import React from "react";
import { mount, shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { UnWrapped } from "./../";

describe("<TextInput />", () => {
  it("render component <TextInput>", () => {
    const wrapper = mount(<UnWrapped />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
