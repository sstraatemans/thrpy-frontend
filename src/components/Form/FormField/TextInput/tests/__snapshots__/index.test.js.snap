exports[`<TextInput /> render component <TextInput> 1`] = `
<TextInput>
  <div>
    <styled.input
      hasPlaceholder={false}
      required="required"
      type="text">
      <input
        className="sc-bdVaJa alSZz"
        required="required"
        type="text" />
    </styled.input>
    <styled.span>
      <span
        className="sc-bwzfXH eLKYQt" />
    </styled.span>
  </div>
</TextInput>
`;
